gather_gitops.sh

```diff
@@ -0,0 +1,198 @@
+#!/bin/bash
```

Comment:
```suggestion
#!/usr/bin/env bash
```

Reason: https://askubuntu.com/questions/1402718/is-the-same-usr-bin-env-bash-than-bin-bash